wildfly/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -1619,7 +1621,12 @@ private void parseEviction(XMLExtendedStreamReader reader, ModelNode cache, Map<
             Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
             switch (attribute) {
                 case STRATEGY: {
-                    enabled = EvictionStrategy.valueOf(value).isEnabled();
+                    try {
+                        enabled = EvictionStrategy.valueOf(value).isEnabled();
+                    } catch (IllegalArgumentException e) {
+                        // Unable to parse eviction strategy - assuming it is enabled
+                        enabled = true;
```

Comment:
We should throw an Exception here if the parser is 11 or higher and the strategy is not recognised.

Reply:
Sure